pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[project]
+name="adios2"
+version="0.0.1"
+description="The Adaptable IO System"
+authors = [
+  {name = "Vicente Adolfo Bolea Sanchez", email = "vicente.bolea@kitware.com"},
+]
+
+keywords = [
+    "Python",
+    "Web",
+    "Application",
+    "Framework",
+]
+classifiers = [
+    "License :: Other/Proprietary License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3 :: Only",
+    "Topic :: Software Development :: Libraries :: Application Frameworks",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+
+dependencies = [
+    "numpy",
+]
+
+[project.optional-dependencies]
+dev = [
+  "pip>=21.3",
+  "pytest",
+  "setuptools",
```

Comment:
Do you know what you're using `setuptools` for?  I don't see a `setup.py` or a `setup.cfg` in the changeset, but still maybe you've imported it specifically somewhere, or just discovered it was needed?

Reply:
hmm actually we do not need pytest we run the tests with `ctest` in the light of it it might be better to use the builtin module to avoid using external pkgs. What do you think?

Reply:
Good catch, yes we do not need setupstools we use cmake for installing the adios2 python package.